src/ansys/motorcad/core/methods/rpc_methods_general.py

```diff
@@ -458,3 +458,45 @@ def quit(self):
     def set_free(self):
         """Free the Motor-CAD instance."""
         return self.connection._set_free()
+
+    def download_mot_file(self, file_path):
+        """Download the current .mot file from Motor-CAD and write it to a local directory.
+
+        This allows users to download .mot files to a local directory from a Motor-CAD instance
+        on a remote machine.
+
+        Parameters
+        ----------
+        file_path : str
+            Full path to the mot file, including the file name and .mot extension.
+            Use the ``r'filepath'`` syntax to force Python to ignore special characters.
+        """
+        self.connection.ensure_version_at_least("2023.2")
+        method = "GetCurrentFileJSON"
+        file_contents = self.connection.send_and_receive(method)
+
+        with open(file_path, "w") as mot_file:
+            for line in file_contents:
+                mot_file.write(line + "\n")
+
+    def upload_mot_file(self, file_path):
+        """Upload a .mot file to Motor-CAD instance from local directory.
+
+        This allows users to send .mot files from a local directory to a Motor-CAD instance
+        on a remote machine.
+
+        Parameters
+        ----------
+        file_path : str
+            Full path to the mot file, including the file name and .mot extension.
+            Use the ``r'filepath'`` syntax to force Python to ignore special characters.
+        """
+        self.connection.ensure_version_at_least("2023.2")
+        file_contents = []
+        with open(file_path, "r") as mot_file:
```

Comment:
What if the mot file doesn't exist? Do we need to show an error to the user?

Reply:
![image](https://github.com/ansys/pymotorcad/assets/80680946/de50b096-336b-43e9-941b-47aba80dd15d)